Description: Disable building the Data/ODBC/SQLExecutor test object on armel.
 It seems to cause a "virtual memory exhausted" error.
Origin: vendor
Forwarded: no
Author: Cristian Greco <cristian@debian.org>

--- a/Data/ODBC/testsuite/Makefile
+++ b/Data/ODBC/testsuite/Makefile
@@ -51,7 +51,11 @@
 
 objects = ODBCTestSuite Driver \
 	ODBCDB2Test ODBCMySQLTest ODBCOracleTest \
-	ODBCPostgreSQLTest ODBCSQLiteTest ODBCSQLServerTest SQLExecutor
+	ODBCPostgreSQLTest ODBCSQLiteTest ODBCSQLServerTest
+
+ifneq ($(POCO_HOST_OSARCH),armel)
+    objects += SQLExecutor
+endif
 
 ifeq ($(POCO_CONFIG),MinGW)
     objects += ODBCAccessTest ODBCSQLServerTest
